qcodes/instrument_drivers/Keysight/keysight_b220x.py

```diff
@@ -0,0 +1,181 @@
+from qcodes import VisaInstrument
+from qcodes.utils.validators import MultiType, Ints, Enum, Lists
+
```

Comment:
Notebook:
For the `couple_port_autodetect` example: for the good example, the output says `set()` while i guess it should be `{(3,4), (7,8)}` according to the comment in the code?